tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = django{18,19,110}
+envlist = django{18,19,110,111}
```

Comment:
1.9 is [no longer a supported release](https://www.djangoproject.com/download/), should we continue to test against it?